debug_toolbar/static/debug_toolbar/js/toolbar.js

```diff
@@ -270,16 +279,25 @@ const djdt = {
                 if (
                     this.getAllResponseHeaders().indexOf("djdt-store-id") >= 0
                 ) {
-                    let store_id = this.getResponseHeader("djdt-store-id");
-                    store_id = encodeURIComponent(store_id);
-                    const dest = `${sidebar_url}?store_id=${store_id}`;
-                    slowjax(dest).then(function (data) {
-                        replaceToolbarState(store_id, data);
-                    });
+                    handleAjaxResponse(this.getResponseHeader("djdt-store-id"));
                 }
             });
             origOpen.apply(this, arguments);
         };
+
+        const origFetch = window.fetch;
+        window.fetch = function (url, options) {
+            const promise = origFetch(url, options);
```

Comment:
It probably makes no sense and it shouldn't happen anyway but it might be more cautious to use `origFetch.apply(this, arguments)` here (similar to the XHR case above)

Apart from that: Looks good to me, thanks!

Reply:
Will do!

Reply:
Done. I'll squash the commits after approval.